whiteboard/whiteboard-ps/whiteboard/Get-Whiteboard.md

```diff
@@ -44,6 +44,37 @@ PS C:\>Get-Whiteboard -UserId 00000000-0000-0000-0000-000000000001
 
 Get all of a user's Whiteboards.
 
+### -------------------------- EXAMPLE 2 --------------------------
+
+```
+PS C:\>Get-Whiteboard -UserId 00000000-0000-0000-0000-000000000001 -WhiteboardId 00000000-0000-0000-0000-000000000002
+```
+
+Get user's specific Whiteboard.
+
+
+### Response
+
+```yaml
+  baseApi                  : baseApi-value
+  id                       : whiteboardId-value
+  userId                   : userId-value
+  folioId                  : folioId-value
+  title                    : title-value
+  createdBy                : createdBy-value
+  ownerId                  : ownerId-value
+  ownerTenantId            : ownerTenantId-value
+  isShared                 : isShared-value
+  createdTime              : createdTime-value
+  invitedTime              : invitedTime-value
+  personalLastModifiedTime : personalLastModifiedTime-value
+  lastModifiedTime         : lastModifiedTime-value
+  globalLastViewedTime     : globalLastViewedTime-value
+  lastViewedTime           : lastViewedTime-value
+  meetingId                : meetingId-value
+  eTag                     : eTag-value
+```
```

Comment:
```suggestion
PS C:\>Get-Whiteboard -UserId 00000000-0000-0000-0000-000000000001 -WhiteboardId 00000000-0000-0000-0000-000000000002

  baseApi                  : baseApi-value
  id                       : whiteboardId-value
  userId                   : userId-value
  folioId                  : folioId-value
  title                    : title-value
  createdBy                : createdBy-value
  ownerId                  : ownerId-value
  ownerTenantId            : ownerTenantId-value
  isShared                 : isShared-value
  createdTime              : createdTime-value
  invitedTime              : invitedTime-value
  personalLastModifiedTime : personalLastModifiedTime-value
  lastModifiedTime         : lastModifiedTime-value
  globalLastViewedTime     : globalLastViewedTime-value
  lastViewedTime           : lastViewedTime-value
  meetingId                : meetingId-value
  eTag                     : eTag-value
```

Get a user's specific Whiteboard.

```

Reply:
@dariomws The reason I didn't add the response to the same cmdlet yaml box was because both the scripts on example 1 and 2 will output the whiteboard(s) with those properties. The first script displays all the whiteboards and second one will just display 1 whiteboard. We wanted to show that these properties are expected.

Reply:
Usually, on Get- cmdlet articles, the output or results are not included. If it is really necessary, I can recall some articles that include them inside the powershell code block.

Reply:
Thanks @kausali for your explanation. It´s something we usually don´t see in other cmdlet articles but I think we can leave it there.